lib/Parse/ParseExpr.cpp

```diff
@@ -1613,6 +1613,9 @@ ParserResult<Expr> Parser::parseExprPrimary(Diag<> ID, bool isExprBasic) {
       return makeParserErrorResult(new (Context) ErrorExpr(DotLoc));
     SyntaxContext->createNodeInPlace(SyntaxKind::MemberAccessExpr);
 
+    auto * UME = new (Context) UnresolvedMemberExpr(DotLoc, NameLoc, Name,
+                                                    /*implicit=*/false);
+
     // Check for a () suffix, which indicates a call when constructing
```

Comment:
Do we really need the rest of this block (L1619-1662) ? I guess `parseExprPostfixSuffix()` can do equivalent.

Reply:
Yeah seems like that should work!